src/integTest/kotlin/io/snyk/plugin/services/SnykTaskQueueServiceTest.kt

```diff
@@ -120,6 +123,20 @@ class SnykTaskQueueServiceTest : LightPlatformTestCase() {
         assertFalse(settings.snykCodeQualityIssuesScanEnable)
     }
 
+    @Test
+    fun `test should disable Code settings when LCE is enabled`() {
+        val snykTaskQueueService = project.service<SnykTaskQueueService>()
+        val settings = pluginSettings()
+        settings.sastOnServerEnabled = true
```

Comment:
This may affect other tests - what do you think? Should we reset the settings to the state before at the end of the test?

Reply:
Clarified, settings will be reset.